Source/VisualRecognitionV3/VisualRecognition.swift

```diff
@@ -457,61 +310,66 @@ public class VisualRecognition {
         request.responseObject(responseToError: responseToError) {
             (response: RestResponse<Classifier>) in
             switch response.result {
-            case .success(let classifier): success(classifier)
+            case .success(let retval): success(retval)
             case .failure(let error): failure?(error)
             }
         }
     }
 
     /**
-     Delete a custom classifier.
+     Delete a classifier.
 
-     - parameter withID: The id of the classifier to delete.
+     - parameter classifierID: The ID of the classifier.
      - parameter failure: A function executed if an error occurs.
-     - parameter success: A function executed after the classifier has been successfully deleted.
-     */
+     - parameter success: A function executed with the successful result.
+    */
     public func deleteClassifier(
-        withID classifierID: String,
+        classifierID: String,
         failure: ((Error) -> Void)? = nil,
```

Comment:
Shouldn't we be marking the `failure` parameter as `@escaping` as well?
> A closure is said to escape a function when the closure is passed as an argument to the function, but is called after the function returns.

Reply:
Hmm, it certainly seems like it should be. But even in the handwritten code we haven't had to mark `failure` with `@escaping`. Pretty sure Xcode would report an error if we didn't mark `success`, but it doesn't seem to mind about `failure`.

I think I'll leave this the way it is for now, but create an issue to look into this in more depth. We've had published code for a long time without `@escaping`, so I don't think it will be a problem.

We might end up moving to a single completion handler before circling back to this.